staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -128,6 +128,15 @@ func NewUnauthorized(reason string) *StatusError {
 
 // NewForbidden returns an error indicating the requested action was forbidden
 func NewForbidden(qualifiedResource schema.GroupResource, name string, err error) *StatusError {
+	var message string
+	res := qualifiedResource.String()
+	if res == "" {
```

Comment:
better to check `qualifiedResource.Empty()` than rely on specific string serialization

Reply:
Done